README.md

```diff
@@ -1,37 +1,38 @@
-# student-dashboard-django
+# my-learning-analytics
```

Comment:
I noticed the github project is already renamed!

Reply:
Yeah, I checked to make sure the old repo works so nothing previously defined will break either. Seems good to me. Maybe now we can stop calling it dashboard. ;)